xml/System.Security.Cryptography.X509Certificates/SubjectAlternativeNameBuilder.xml

```diff
@@ -18,7 +18,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This class facilitates building a subject alternative names extension for an X.509 certificate.</summary>
```

Comment:
"names" (plural) should be "name" (singular).  Other references seem to be singular.